### PR TITLE
#88 send qr id and access token to backend

### PR DESCRIPTION
**Pull request by hexxunset**

### CODE REVIEW

**Review by Burgurrd — DISMISSED** [review]
Looks good.

File: backend/src/main/kotlin/com/ntnu/backend/controller/KafkaController.kt [hunk 1]
```diff
@@ -19,10 +20,17 @@ class KafkaController(val kafkaTemplate: KafkaTemplate<String, String>, properti
...
     @ResponseStatus(HttpStatus.CREATED)
-    fun sendApplication(@RequestHeader(name="Authorization") token: String, @RequestBody questionnaireResponseId: String): ResponseEntity<String> {
-        val response = khttp.get("http://launch.smarthealthit.org/v/r3/fhir/QuestionnaireResponse/${questionnaireResponseId}", headers = mapOf("Authorization" to token, "Content-Type" to "application/fhir-json"))
+    fun sendApplication(@RequestHeader(name="Authorization") token: String, @RequestBody body: String): ResponseEntity<String> {
+        var b1 = body
+        if (body.takeLast(1) == "=") {
+            val b1 = body.dropLast(1)
+        }
+        println("Token:${token}")
+        println("B1:${b1}")
```
Comment (agnarbjoernstad) [inline backend/src/main/kotlin/com/ntnu/backend/controller/KafkaController.kt]: Remove println.

**Review by agnarbjoernstad — CHANGES_REQUESTED** [review]
Remove println and console.log.

**Review by Burgurrd — DISMISSED** [review]
Looks good.

**Review by Burgurrd — APPROVED** [review]
All good.

**Review by agnarbjoernstad — APPROVED** [review]

**Pull request merged**